recommender/recommender.py

```diff
@@ -657,43 +678,106 @@ def endorse_resource(self, data, _suffix=''):
     @XBlock.json_handler
     def deendorse_resource(self, data, _suffix=''):
```

Comment:
If you're changing the wording inside the function from "deendorse" to "remove", then shouldn't you change the function name, too?